Remove links to istio-ecosystem. (#5223)
istio/istio.io: content/en/blog/2019/isolated-clusters/index.md

## The current mesh-federation work

While you can perform mesh federation using standard Istio configurations already today,

> it requires writing a lot of boilerplate YAML files and is error-prone. There is an experimental effort

Reviewer: ```suggestion
it requires writing a lot of boilerplate YAML files and is error-prone. There is an effort under way to automate
the mesh federation process. In the meantime, you can look at this
[multi-mesh deployment example](https://github.com/istio-ecosystem/multi-mesh-examples)
to get an idea of what kind of artifacts a generated federation might include.
```

> under way to [automate the mesh federation process](https://github.com/istio-ecosystem/multi-mesh-examples).

Reviewer: ```suggestion
```

> If this experiment pans out, it may be folded into Istio proper at some point.

Reviewer: ```suggestion
```
